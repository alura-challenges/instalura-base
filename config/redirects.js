module.exports = [
  {
    source: '/login/',
    destination: '/app/login/',
    permanent: true, // 301 == 308 || 302 == 307
  },
];
