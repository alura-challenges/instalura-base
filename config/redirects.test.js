import redirects from './redirects';

// Test Driven Development
describe('config/redirects', () => {
  test('renders all current redirects', () => {
    expect(redirects).toMatchSnapshot(); // Fotografia/Insta/Polaroid
  });
});
