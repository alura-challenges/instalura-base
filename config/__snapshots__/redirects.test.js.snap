// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/redirects renders all current redirects 1`] = `
Array [
  Object {
    "destination": "/app/login",
    "permanent": true,
    "source": "/login/",
  },
]
`;
